src/script/server/api/services/state/service.js

```diff
@@ -1,11 +1,11 @@
 const { NAME } = require('./constants')
-const db = require('../../db')(NAME)
+const db = require('../../db')
 
 const create = async (state) => {
-  await db.append(state)
+  await db(NAME).write(state)
```

Comment:
I have to call the factory there (for each request), because redux state have to be set (and updated in further updates)